README.md

```diff
@@ -1,18 +1,45 @@
-<h1 align="center">Invidious</h1>
+<div align="center">
+  <img src="assets/android-chrome-192x192.png">
+  <h1>Invidious</h1>
+
+  <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">
+    <img alt="License: AGPLv3+" src="https://shields.io/badge/License-AGPL%20v3+-blue.svg">
+  </a>
+  <a href="https://github.com/iv-org/invidious/actions">
+    <img alt="Build Status" src="https://github.com/iv-org/invidious/workflows/Invidious%20CI/badge.svg">
+  </a>
+  <a href="https://github.com/iv-org/invidious/issues">
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/iv-org/invidious?color=important">
+  </a>
+  <a href="https://github.com/iv-org/invidious/issues">
+    <img alt="GitHub pull requests" src="https://img.shields.io/github/issues-pr/iv-org/invidious?color=blueviolet">
+  </a>
+  <a href="https://hosted.weblate.org/engage/invidious/">
+    <img alt="Translation Status" src="https://hosted.weblate.org/widgets/invidious/-/translations/svg-badge.svg">
+  </a>
+
+  <h3>An open source alternative front-end to YouTube</h3>
+
+  <a href="https://instances.invidious.io/">Instances list</a>
+  &nbsp;•&nbsp; 
+  <a href="#documentation">Documentation</a>
+  &nbsp;•&nbsp;
+  <a href="#contribute">Contribute</a>
+  &nbsp;•&nbsp;
+  <a href="#donate">Donate</a>
+
+  <p>Chat with us on:</p>
```

Comment:
Make this a header, that way we can link directly to it (like that https://github.com/iv-org/invidious/blob/eb97e584d0e4e3d47444d0f6d4aa76736131ebf8/README.md#an-open-source-alternative-front-end-to-youtube )

Reply:
Sure!